include/mata/inter-aut.hh

```diff
@@ -0,0 +1,193 @@
+//
+// Created by Martin Hruska on 09.08.2022.
+//
+
+#ifndef _MATA_INTER_AUT_HH
+#define _MATA_INTER_AUT_HH
+
+#include <unordered_map>
+#include <unordered_set>
+
+#include <mata/parser.hh>
+
+#include <string>
+#include <vector>
+
+namespace Mata
+{
+
+/**
+ * A node of graph representing transition formula. A node could be operator (!,&,|) or operand (symbol, state, node).
+ * Each nodes has a name (in case of marking naming, an initial character definig type of node is removed and stored in
+ * name), raw (name including potential type marker), and information about its type.
+ */
+struct FormulaNode
+{
+public:
+    enum OperandType {
+        SYM,
```

Comment:
Maybe rename to `SYMBOL`? Those 3 chars are hardly a shortening.

Reply:
`SYMBOL` would be even better in my opinion

Reply:
I prefer abbreviations however, in this case it may be better.